djautotask/utils.py

```diff
@@ -2,17 +2,26 @@
 
 
 class DjautotaskSettings:
+    force_batch_query_size = 215
 
     def get_settings(self):
         request_settings = {
             'timeout': 30.0,
             'batch_size': 50,
             'max_attempts': 3,
             'keep_completed_hours': 8,
-            'batch_query_size': 400,
+            'batch_query_size': self.force_batch_query_size,
         }
 
+        # service_call_task shows errors w/ the value of batch_query_size
+        # larger than 215. The error: 404 - File or directory not found.</h2>
+        # <h3>The resource you are looking for might have been removed,
+        # had its name changed, or is temporarily unavailable.
         if hasattr(settings, 'DJAUTOTASK_CONF_CALLABLE'):
             request_settings.update(settings.DJAUTOTASK_CONF_CALLABLE())
+            request_settings['batch_query_size'] = min(
+                self.force_batch_query_size,
+                request_settings.get('batch_query_size')
+            )
```

Comment:
I don't think any of this should be in `DjautotaskSettings`. If I am reading this correctly it will limit ALL batch queries to 215, even the ones that would work with a higher number.

Instead you should just have `ServiceCallTaskSynchronizer` override its __init__ method, and then you can just set `self.batch_query_size` to be 215 there with this `min` method.

Reply:
That's a good point. Will fix that.